Resources/Prototypes/Nuclear14/Recipes/Ammunition/AmmunationRecycling.yml

```diff
@@ -0,0 +1,54 @@
+- type: Construction
+ name: .308 round casing
+ id: 308roundCasing
+ graph: 308roundCasing
+ startNode: start
+ targetNode: 308roundCasing
+ category: Ammo
+ description: The casings of .308 rounds
+ icon: 
+  sprite: Objects/Weapons/Guns/Ammunition/Casings/ammo_casing.rsi #Change as necessary
+  
+- type: Construction
```

Comment:
`construction` instead of `Construction` as it fits with the rest of the data better